sign_oca/__manifest__.py

```diff
@@ -5,14 +5,16 @@
     "name": "Sign Oca",
     "summary": """
         Allow to sign documents inside Odoo CE""",
-    "version": "18.0.1.0.0",
+    "version": "18.0.2.0.0",
```

Comment:
```suggestion
    "version": "18.0.2.0.0",
```

This is not necessary (only if a migration script is added), it will be done automatically by the bot when merging.

Reply:
done